source/includes/_files.md

```diff
@@ -0,0 +1,199 @@
+# Files
+
+On the OpenDataSoft platform, files can be used as data source to create new datasets.
```

Comment:
It can be used as attachment.
Maybe more context, like: to create a dataset from a file, the first thing to do is to upload the file to the platform ? @meganelacheny what do you think ? Is it useful ?